src/annotator/anchoring/text-range.ts

```diff
@@ -76,8 +76,14 @@ function resolveOffsets(element, ...offsets) {
   return results;
 }
 
-export let RESOLVE_FORWARDS = 1;
-export let RESOLVE_BACKWARDS = 2;
+/**
+ * When resolving a TextPosition, specifies the direction to search for the
+ * nearest text node if `offset` is `0` and the element has no text.
+ */
+export enum ResolveDirection {
```

Comment:
We haven't made much use of `enum` but it seemed to fit the bill here.

Reply:
We didn't use `enum` in JSDoc because it is a TS-specific thing which requires TS or Babel to produce code at runtime, unlike almost all other TS constructs. In modern TS there is an alternative using [plain objects](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums), but it is less convenient to use.